index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img id="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <a href=""
+        ><img
+          id="nav-logo"
+          src="./images/header/Hamburger_icon.svg.png"
+          alt="logo"
+      /></a>
+    </header>
+    <main id="headline">
+      <section>
+        <div id="name"><p>I’m Ali Hassan. Glad to see you!</p></div>
+        <div id="introduction">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Repudiandae, maiores vero nesciunt perferendis non, quae est fugiat
+            labore ratione fuga quam. Nesciunt voluptas neque nisi veritatis?
+            Officia dolore maxime explicabo.
+          </p>
```

Comment:
- [ ] You did great using the p tag here. :+1:  This section briefly tells the user or your future employer a bit about you, this is your portfolio. You should not just use placeholder texts, you should use the real content provided alongside the Figma design, I'll suggest you go to the Figma design and the correct text for this section. :+1: 

- - THIS SHOULD BE THE CORRECT TEXT:
![Screenshot from 2023-06-02 18-09-44](https://github.com/mhassa204/Portfolio-Project/assets/107632780/c0a487d0-6230-43c8-9b65-4d6cfd57745d)